src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -15,6 +15,8 @@ namespace Xamarin.Android.Build.Tests
 {
 	public class BaseTest
 	{
+		public static Dictionary<string, string> TestOutputDirectories = new Dictionary<string, string> ();
```

Comment:
If tests are running in parallel, I think we need a lock or `ConcurrentDictionary`, etc.